CHANGES.rst

```diff
@@ -1,11 +1,12 @@
 Changes
 =======
 
-Version 0.9.1 (UNRELEASED)
+Version 0.9.1 (2023-09-27)
 --------------------------
 
-- Fixes the total job count reported to workflow controller by excluding jobs restored from the workspace and not executed by Snakemake.
+- Fixes the reported total number of jobs by excluding cached jobs that are restored from the workspace and not executed by Snakemake.
```

Comment:
- Fixes the reported total number of jobs for restarted workflows by excluding cached jobs that were simply reused from previous runs in the workspace and not really executed by Snakemake.